sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -304,7 +304,40 @@ def test_piecewise_solve():
                   (-x + 2, x - 2 <= 0), (x - 2, x - 2 > 0))
     assert solve(g, x) == [5]
 
-# See issue 4352 (enhance the solver to handle inequalities).
+
+def test_piecewise_solveset():
```

Comment:
I found test case for `solve`  just [above](https://github.com/sympy/sympy/pull/10550#discussion-diff-52560677R279) so copied the same cases for solveset to check it is working correctly or not.
I have shifted the both testcases.